conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt

```diff
@@ -6,38 +6,19 @@ import java.io.Serializable
 /**
  * Default model for a user.
  */
-class DefaultUser: IUser, Serializable {
-    private var id: String
-    private var name: String?
-    private var avatar: String?
-
-    constructor(id: String, name: String?, avatar: String?) {
-        this.id = id
-        this.name = name
-        this.avatar = avatar
-    }
-
-    fun setId(id: String) {
-        this.id = id
-    }
+data class DefaultUser(val userId: String,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/itsAlexNguyen/Conveyor/blob/e9a571e2f03220124bea1aedfe5c6054655da1e0/conveyor/src/main/java/com/hodamohammadi/conveyor/models/DefaultUser.kt#L9"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>